content/docs/dvclive/api-reference/live/log_image.md

```diff
@@ -30,44 +30,27 @@ Supported values for `val` are:
   [PIL.Image.fromarray](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.fromarray))
 - A `PIL.Image` instance.
 
-The images will be saved in `{Live.dir}/images/{name}`:
+The images will be saved in `{Live.plots_dir}/images/{name}`:
 
 ```dvc
 $ tree
```

Comment:
💅🏼 Also, no need to show the `tree` command IMO (not available on Windows by default, maybe same with Mac). It's clear that this is a folder structure, I think.